File: submissions/2767-maximum-sum-with-exactly-k-elements-/solution.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var maximizeSum = function(nums, k) {
    let max = nums[0];
    for(let x of nums) max = Math.max(...nums);
    let sum = max * k + (k-1)*(k)/2;
    return sum;
};

File: submissions/2767-maximum-sum-with-exactly-k-elements-/solution.java
class Solution {
    public int maximizeSum(int[] nums, int k) {
        int n = nums.length;
        int max = Integer.MIN_VALUE;
        for(int x : nums) max = Math.max(max, x);
        int sum = max * k + (k-1)*(k)/2;
        return sum;
    }
}
